package.json

```
@@ -10,14 +10,17 @@
     "build": "turbo build",
     "deploy:aa": "turbo deploy --filter=antalmanac",
     "deploy:cdk": "turbo deploy --filter=cdk",
-    "deploy": "turbo deploy"
+    "deploy": "turbo deploy",
+    "test": "vitest run",
+    "test:watch": "vitest"
```

For simplicity, I think I'll do this for the lightest abstraction; if you want to specifically invoke `vitest run`, then you can do `pnpm test run`

```suggestion
    "deploy": "turbo deploy",
    "test": "vitest"
```